xmtp_api_d14n/src/protocol/extractors/topics.rs

```diff
@@ -86,11 +89,33 @@ impl EnvelopeVisitor<'_> for TopicExtractor {
         Ok(())
     }
 
+    fn visit_welcome_message_version(
```

Comment:
the matching here should be done by the implementation on [`welcome_message_input::Version`](https://xmtp.github.io/libxmtp/xmtp_api_d14n/protocol/traits/trait.ProtocolEnvelope.html#impl-ProtocolEnvelope%3C'env%3E-for-Version-2), so I think we can remove this